src/applications/ivc-champva/shared/components/applicantLists/CustomCheckboxRadioPage.jsx.jsx

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
Made this custom component because standard checkbox configs don't support dynamic label text (e.g., showing current list loop element's name in the labels). This can/will be reused across this form (and potentially other forms in the IVC group)